<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>810</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Лабораторная работа №2, Вариант 6, Соколов Егор, гр 143.</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelTask">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>21</y>
      <width>571</width>
      <height>181</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resourses.qrc">:/images/images/task.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelVariantTask">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>571</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resourses.qrc">:/images/images/variant_task.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frameDefine">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>561</width>
      <height>121</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frameFind">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>160</y>
      <width>631</width>
      <height>381</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableWidget" name="tableWidgetResult">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>611</width>
       <height>261</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>1) Метод дихотомии</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2) Метод пропорциональных частей (хорд)</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3) Метод касательных (Ньютона)</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4) Метод Ньютона модифицированный</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>5) Комбинированный метод</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>6) Итерационный метод</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>x</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>f(x)</string>
      </property>
     </column>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>15</y>
       <width>611</width>
       <height>82</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButtonStart">
        <property name="text">
         <string>Уточнить корень</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="2">
         <widget class="QLineEdit" name="lineEditB"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelInfoEPS">
          <property name="text">
           <string>Погрешность eps =</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="lineEditA">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelInfoB">
          <property name="text">
           <string>Правая граница b = </string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelInfoA">
          <property name="text">
           <string>Левая граница a = </string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="lineEditEps">
          <property name="text">
           <string>0.0001</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="labelCheckName">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>260</y>
      <width>301</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Контрольный расчёт</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCheck">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>392</width>
      <height>180</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resourses.qrc">:/images/images/smath.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>30</y>
      <width>541</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1" rowspan="2">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QLabel" name="labelDefineA">
         <property name="text">
          <string>Левая граница a = </string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="lineEditDefineStep">
         <property name="text">
          <string>0.1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="lineEditDefineB">
         <property name="text">
          <string>100</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="lineEditDefineA">
         <property name="text">
          <string>-100</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="labelDefineB">
         <property name="text">
          <string>Правая граница b = </string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="labelDefineStep">
         <property name="text">
          <string>Шаг для отделения =</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButtonDefineRoots">
         <property name="text">
          <string>Отделить корень</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="labelCheck_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>490</y>
      <width>451</width>
      <height>271</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resourses.qrc">:/images/images/matplotlib.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="action_clean"/>
   </widget>
   <widget class="QMenu" name="menu_info">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_info"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_clean">
   <property name="text">
    <string>Очистить поля</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>lineEditDefineA</tabstop>
  <tabstop>lineEditDefineB</tabstop>
  <tabstop>lineEditDefineStep</tabstop>
  <tabstop>pushButtonDefineRoots</tabstop>
  <tabstop>lineEditA</tabstop>
  <tabstop>lineEditB</tabstop>
  <tabstop>pushButtonStart</tabstop>
  <tabstop>tableWidgetResult</tabstop>
 </tabstops>
 <resources>
  <include location="resourses.qrc"/>
 </resources>
 <connections/>
</ui>
